packages/world/mud.config.ts

```diff
@@ -33,21 +33,21 @@ export default mudConfig({
         moduleName: "bytes16",
         argumentsHash: "bytes32", // Hash of the params passed to the `install` function
       },
-      schema: {
+      valueSchema: {
         moduleAddress: "address",
       },
       // TODO: this is a workaround to use `getRecord` instead of `getField` in the autogen library,
-      // to allow using the table before it is registered. This is because `getRecord` passes the schema
+      // to allow using the table before it is registered. This is because `getRecord` passes the valueSchema
       // to store, while `getField` loads it from storage. Remove this once we have support for passing the
-      // schema in `getField` too. (See https://github.com/latticexyz/mud/issues/444)
+      // valueSchema in `getField` too. (See https://github.com/latticexyz/mud/issues/444)
```

Comment:
I think `getField` now supports passing in value schema (or field layout now) so is this comment/workaround still relevant?

Reply:
true, will remove this in a followup